City-of-Helsinki/drupal-helfi-sote
# UHF-X Top bar & updating the project to latest configuration

To test:

1. Checkout the branch on this repo and also checkout the hdbt repo to a branch that has the same name.
2. `make new`
3. Login to the site and go to add some links to header top navigation: https://helfi.docker.sh/fi/admin/structure/menu/manage/header-top-navigation. Add one internal and one external so that you can see that both work ok.
4. Check that the top bar looks the way it should in the screenshots.
5. Additional stuff: Try out the site like for example try to add unit search to landing pages etc since a lot of configuration has been updated.

Checkout also these:
https://github.com/City-of-Helsinki/drupal-helfi/pull/121
https://github.com/City-of-Helsinki/drupal-hdbt/pull/93
